From 1cdd375c7967d737d22942cd4e1921f2ab1d7819 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Fri, 2 Jun 2017 17:12:11 +0200
Subject: [PATCH 4/5] RunGrepTest: skip script callout test

The problem with this test is that it expects a Unix setup (e.g.
/bin/echo to be present), which is not the case in MSYS2.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 RunGrepTest | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/RunGrepTest b/RunGrepTest
index 3345c7b..cc42509 100755
--- a/RunGrepTest
+++ b/RunGrepTest
@@ -971,6 +971,7 @@ fi
 # environments that do not support fork(). This is handled by comparing to a
 # different output.
 
+cat >/dev/null <<\SKIP_ON_WINDOWS
 if $valgrind $vjs $pcre2grep --help | $valgrind $vjs $pcre2grep -q 'callout scripts in patterns are supported'; then
   echo "Testing pcre2grep script callouts"
   $valgrind $vjs $pcre2grep '(T)(..(.))(?C"/bin/echo|Arg1: [$1] [$2] [$3]|Arg2: $|${1}$| ($4) ($14) ($0)")()' $srcdir/testdata/grepinputv >testtrygrep
@@ -1006,6 +1007,7 @@ if $valgrind $vjs $pcre2grep --help | $valgrind $vjs $pcre2grep -q 'callout scri
 else
   echo "Script callouts are not supported"
 fi
+SKIP_ON_WINDOWS
 
 
 # Test reading .gz and .bz2 files when supported.
-- 
2.39.0.rc0.windows.1

